sdk/python/touca/cli/_post.py

```diff
@@ -90,6 +108,15 @@ def run(self):
         except ValueError as err:
             print(err, file=sys.stderr)
             return False
+            
+        if dry_run:
+            logger.info(f"dry run is enabled")
+            for batchName in batchNames:
+                batchDir = src_dir.joinpath(batchName)
+                if not dry_run_post(batchDir):
+                    logger.error(f"unable to post {batchDir}")
+                    return False
+            return True
```

Comment:
This is nitpick but wouldn't it make more sense to the reader to see `dry_run_post` take `batchNames` and take care of everything?
```suggestion
        if dry_run:
            return dry_run_post(batchNames)
```

Reply:
I can do that. I was more or less mimicking the format from 121-128, but this makes sense